src/Arcus.Scripting.Sql/Scripts/Invoke-AzSqlDatabaseMigration.ps1

```diff
@@ -0,0 +1,154 @@
+param(
+    [parameter(Mandatory=$true)][string] $ServerName = $(throw "Please provide the name of the SQL Server that hosts the SQL Database. (Do not include 'database.windows.net'"),
+    [parameter(Mandatory=$true)][string] $DatabaseName = $(throw "Please provide the name of the SQL Database"),
+    [parameter(Mandatory=$true)][string] $UserName = $(throw "Please provide the UserName of the SQL Database"),
+    [parameter(Mandatory=$true)][string] $Password = $(throw "Please provide the Password of the SQL Database"),
+    [parameter(Mandatory=$false)][string] $ScriptsFolder = "$PSScriptRoot/sqlScripts",
+    [parameter(Mandatory=$false)][string] $ScriptsFileFilter = "*.sql",
+    [parameter(Mandatory=$false)][string] $DatabaseSchema = "dbo"
+)
+
+Write-Host "Looking for SQL scripts in folder: $ScriptsFolder"
+
+#Functions for repeated use
+function Execute-DbCommand($params, [string]$query)
+{
+    $result = Invoke-Sqlcmd @params -Query $query -Verbose -QueryTimeout 180
+    Write-Host "Result from querying $result"
+}
+
+function Execute-DbCommandWithResult($params, [string] $query)
+{
+    $result = Invoke-Sqlcmd @params -Query $query -Verbose
+    return $result
+}
+
+function Create-DbParams([string] $DatabaseName, [string] $serverInstance, [string] $UserName, [string] $Password)
+{
+    Write-Host "databasename = $DatabaseName"
+    Write-Host "serverinstance = $serverInstance"
+    Write-Host "username = $UserName"
+    
+    return $params = @{
+      'Database' = $DatabaseName
+      'ServerInstance' = "$serverInstance.database.windows.net"
+      'Username' = $UserName
+      'Password' = $Password
+      'OutputSqlErrors' = $true
+      'AbortOnError' = $True
+    }
+}
+
+function Get-SqlScriptFileText([string] $scriptPath, [string] $fileName)
+{
+    $currentfilepath = "$scriptPath/$fileName.sql"
+    return $query = Get-Content $currentfilepath
+}
+
+function Get-CurrentDbVersionNumber () {
+    $selectDbVersionScript = 'SELECT TOP 1 CurrentVersionNumber FROM DatabaseVersion ORDER BY CurrentVersionNumber DESC'
```

Comment:
We should take into account the databaseSchema when referring to the DatabaseVersion-table to ensure this is always working successfully.

```suggestion
    $selectDbVersionScript = "SELECT TOP 1 CurrentVersionNumber FROM [$DatabaseSchema].[DatabaseVersion] ORDER BY CurrentVersionNumber DESC"
```